core/trino-spi/src/main/java/io/trino/spi/security/AccessDeniedException.java

```diff
@@ -643,6 +643,11 @@ public static void denyExecuteFunction(String functionName)
         throw new AccessDeniedException(format("Cannot execute function %s", functionName));
     }
 
+    public static void denyExecuteFunction(String functionName, String extraInfo)
```

Comment:
should we also pass function kind?

Reply:
Are you thinking that we should change the method to be:

```
    public static void denyExecuteFunction(String functionName, String functionKind, String extraInfo)
    {
        throw new AccessDeniedException(format("Cannot execute %s function %s%s", functionKind, functionName, formatExtraInfo(extraInfo)));
    }
```

Reply:
Yes.

Reply:
Yes, I guess that makes sense.  To look normal it would have to be formatted as `functionKind.toLowerCase(ROOT)`.  And I probably want to pass the `FunctionKind` enum instance, not a string.

Changed as proposed.